src/main.py

```diff
@@ -78,10 +79,28 @@ def main() -> None:
 
     tv1 = TrackView(Track.from_points(tracks[1]["points"]))
     tv2 = TrackView(Track.from_points(tracks[3]["points"]))
+    tv3 = TrackView(Track.from_points(tracks[4]["points"]))
+
+    track_options = {
+        "Track1": Action(ActionType.CHANGE_VIEW, 2),
+        "Track2": Action(ActionType.CHANGE_VIEW, 3),
+        "Track3": Action(ActionType.CHANGE_VIEW, 4),
+    }
+    track_images = [
+        pygame.image.load("resources/graphics/track1.jpg"),
+        pygame.image.load("resources/graphics/track2.jpg"),
+        pygame.image.load("resources/graphics/track3.jpg")
+    ]
+    options_menu = OptionsMenu(tracks=track_options, tracks_thumbnails=track_images)
+    options_menu.background_image = pygame.image.load(
+        "resources/graphics/menu-background.png"
+    )
 
     window.add_view(menu, 0, True)
-    window.add_view(tv1, 1)
-    window.add_view(tv2, 2)
+    window.add_view(tv1, 2)
+    window.add_view(tv2, 3)
+    window.add_view(tv3, 4)
+    window.add_view(options_menu, 1)
```

Comment:
Previews for second and third tracks are not corresponding to actual tracks. Please fix it and add the forth track also. Furthermore, we can give them nicer names like "DNA", "Straight", "Loop" and so on.

Reply:
To be fixed when last two tracks will be merged